src/msw/filedlg.cpp

```diff
@@ -1028,6 +1028,11 @@ wxFileDialog::wxFileDialog(wxWindow *parent,
 
 {
     // NB: all style checks are done by wxFileDialogBase::Create
+}
+
```

Comment:
```suggestion
    m_data = NULL;
}

```
It is not initialized as NULL.

Reply:
Oh, thanks for noticing this, cherry pick removed this accidentally as it doesn't exist in master. Fixed.